code/dqe/symfilecheck.q

```diff
@@ -0,0 +1,6 @@
+\d .dqe
+hdbdir:@[value;`hdbdir;`hdb]
```

Comment:
should this be an hsym i.e. `` `:hdb``? also should it be defaulting to a TorQ env var perhaps, e.g. ``getenv`KDBHDB`` ?

Reply:
I have `.dqe.hdbdir` set as `getenv`KDBHDB` in `config/settings/dqe.q` and will fix that typo